locales/de.json

```diff
@@ -862,145 +862,145 @@
-  "app.privacy.contactUs.questings": "If you have any questions about this Privacy Policy, please contact us.",
-  "app.privacy.contactUs.title": "Contact Us",
+  "app.privacy.accessInfo.content": "Sie haben das Recht anzufragen, dass das Unternehmen Ihnen Auskunft über gewisse Informationen über unsere Sammlung und Verwendung Ihrer persönlichen Informationen aus den letzten 12 Monaten erteilt. Sobald wir Ihre verifizierbare Verbraucheranfrage erhalten und bestätigt haben, (wie beschrieben in dem Abschnitt “Ausübung von Rechten zu Zugriff, Datenauskunft und Löschung”), werden wir Ihnen über Folgendes Auskunft erteilen:",
+  "app.privacy.accessInfo.disclose.ourPurpose": "Unser Geschäfft oder kommerzieller Zweck für die Sammlung oder den Verkauf Ihrer persönlichen Informationen.",
+  "app.privacy.accessInfo.disclose.personalInfo": "Die Kategorien von persönlichen Informationen, die wir über Sie erfasst haben.",
+  "app.privacy.accessInfo.disclose.piecesInfo": "Die spezifischen Einheiten persönlicher Informationen, die wie über Sie erfasst haben (auch Antrag auf Datenauskunft genannt)",
+  "app.privacy.accessInfo.disclose.soldInfo.disclosures": "Auskünfte für geschäfftliche Zwecke, mit Identifizierung der Kategorien persönlicher Informationen, welche jede Empfänger-Kategorie jeweils erhalten hat",
+  "app.privacy.accessInfo.disclose.soldInfo.sales": "Verkäufe, mit Identifizierung der Kategorien persönlicher Informationen, welche jede Empfänger-Kategorie jeweils erworben hat",
+  "app.privacy.accessInfo.disclose.soldInfo.title": "Falls wir Ihre persönlichen Informationen verkauft haben oder für einen geschäftlichen Zweck bekanntgegeben haben, zwei separate Listen mit Angaben zu:",
+  "app.privacy.accessInfo.disclose.sources": "Die Kategorien der Quellen von persönlichen Informationen, die wir über Sie erfasst haben.",
+  "app.privacy.accessInfo.disclose.thirdParties": "Die Kategorien Dritter, mit welchen wir diese persönlichen Informationen geteilt haben.",
+  "app.privacy.accessInfo.title": "Zugriff zu Rechten für Spezifische Informationen und Datenauskunft",
+  "app.privacy.collectAndUse.advertising": "Wir können dritte Werbe-Unternehmen verwenden, um Ihnen Werbung anzuzeigen, wenn Sie die Seite besuchen. Diese Unternehmen können aggregierte Informationen (ausgeschlossen Ihr Name, Ihre Adresse, Email-Adresse oder Telefonnummer) über Ihre Besuche auf dieser und anderen Seiten verwenden, um Ihnen Werbung zu Waren und Diensten, welche für Sie interessant sind, bereitstellen zu können. Wenn Sie mehr Informationen über diese Tätigkeit haben möchten und um über Ihre Entscheidungsmöglichkeiten darüber, dass diese Informationen nicht von diesen Unternehmen verwendet werden sollen, bescheid wissen möchten,",
```

Comment:
I will keep "aggregierte" in this context, because I think that it is more accurate than "gesammelte". The point here is not just that they've been collected, but that there is a relationship between the pieces of information.

Reply:
I disagree with that. 
Form my point of view relationships are part of the data, so they would not justify using a different word. 
Additionally I think they are [synonyms](https://www.duden.de/synonyme/aggregieren) and I don't feel like they have different meanings. But that might just be my sprachgefühl (yes, according to [dict](https://www.dict.cc/?s=sprachgef%C3%BChl) that is an english word).